src/Microsoft.AspNetCore.Cors/Infrastructure/CorsPolicyBuilder.cs

```diff
@@ -148,14 +148,37 @@ public CorsPolicyBuilder AllowAnyHeader()
         /// </summary>
         /// <param name="preflightMaxAge">A positive <see cref="TimeSpan"/> indicating the time a preflight
         /// request can be cached.</param>
-        /// <returns></returns>
+        /// <returns>The current policy builder</returns>
         public CorsPolicyBuilder SetPreflightMaxAge(TimeSpan preflightMaxAge)
         {
             _policy.PreflightMaxAge = preflightMaxAge;
             return this;
         }
 
         /// <summary>
+        /// Sets the specified <paramref name="isOriginAllowed"/> for the underlying policy.
+        /// </summary>
+        /// <param name="isOriginAllowed">The function used by the policy to evaluate if an origin is allowed.</param>
+        /// <returns>The current policy builder</returns>
+        public CorsPolicyBuilder SetIsOriginAllowed(Func<string, bool> isOriginAllowed)
+        {
+            _policy.IsOriginAllowed = isOriginAllowed;
+            return this;
+        }
+
+        /// <summary>
+        /// Sets the <see cref="CorsPolicy.IsOriginAllowed"/> property of the policy to be configured with 
+        /// a function that allows origins to match configured wildcarded domain when evaluating if the 
```

Comment:
Change to: `... allows origins to match a configured wildcard domain when evaluating ...`